cmd/nerdctl/run_cgroup_linux_test.go

```diff
@@ -239,3 +258,20 @@ func TestRunBlkioWeightCgroupV2(t *testing.T) {
 	base.Cmd("run", "--rm", "--blkio-weight", "300", "-w", "/sys/fs/cgroup", testutil.AlpineImage,
 		"cat", "io.bfq.weight").AssertOutExactly("default 300\n")
 }
+
+func TestRunBlkioWeightCgroupV1(t *testing.T) {
```

Comment:
This won't work on kernel >= 5.10, so probably we don't need this test
https://github.com/torvalds/linux/commit/f382fb0bcef4c37dc049e9f6963e3baf204d815c

Reply:
(Why isn't CI failing?)

Reply:
Opened issue #1200 for tracking the CI issue